src/operator/swapaxis.cc

```diff
@@ -53,20 +53,26 @@ MXNET_REGISTER_OP_PROPERTY(SwapAxis, SwapAxisProp)
 
 Examples::
 
-  x = [[1, 2, 3]])
-  swapaxes(x, 0, 1) = [[ 1],
-                       [ 2],
-                       [ 3]]
+  >>> x = mx.nd.array([[1, 2, 3]]) //(1,3) array
```

Comment:
This comment breaks things. I'd just get rid of it... or put the comment on it's own line.